common/sagas/rates.js

```diff
@@ -2,18 +2,28 @@
 import { put, call } from 'redux-saga/effects';
 import type { Effect } from 'redux-saga/effects';
 import { setRates } from 'actions/rates';
+import { showNotification } from 'actions/notifications';
 
 const symbols = ['USD', 'EUR', 'GBP', 'BTC', 'CHF', 'REP'];
 
-function fetchRates(symbols) {
-  return fetch(
+const fetchRates = symbols =>
+  fetch(
     `https://min-api.cryptocompare.com/data/price?fsym=ETH&tsyms=${symbols.join(
       ','
     )}`
-  ).then(r => r.json());
-}
+  ).then(response => {
```

Comment:
Would prefer adding this to `api/utils/checkHttpStatus.js` and possibly renaming to `ensureOKResponse`. 

Perhaps we can standardize on always chaining `ensureOKResponse` once that is done.